frontend/src/scenes/session-recordings/filters/SimpleSessionRecordingsFilters.tsx

```diff
@@ -47,7 +50,7 @@ export const SimpleSessionRecordingsFilters = ({
 
     return (
         <div className="space-y-2">
-            <div className="flex space-x-1">
+            <div className="flex space-x-1 px-3 overflow-x-scroll pb-3">
```

Comment:
should this have `overflow-x-scroll` or should it wrap?

![2023-08-17 17 45 10](https://github.com/PostHog/posthog/assets/984817/43543bfb-cb7e-482e-8541-1de2608ff070)

Reply:
Great shout! Wrapping is much nicer